boot/zephyr/boards/nrf52_minimal_footprint.conf

```diff
@@ -0,0 +1,39 @@
+# Minimal MCUBoot flash footprint configuration
+# for nRF52832 SoC targets
+# This is not recomendet configuration because of security and reliability
+# reasons.
+
```

Comment:
You are missing quite a few from:
https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/basic/minimal/common.conf
and
https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/basic/minimal/no-timers.conf

Reply:
#CONFIG_SYS_CLOCK_EXISTS=y is required for reboot routine.

I added the rest form minimal - it has no impact on footprint as configuration was by default the same or an API was no used.

Reply:
> #CONFIG_SYS_CLOCK_EXISTS=y is required for reboot routine.

Really? That's a shame, are you sure we cannot reboot without a timer? Seems counter intuitive to me.

> I added the rest form minimal - it has no impact on footprint as configuration was by default the same or an API was no used.

It has no impact on the footprint? That's strange, there were a bunch there that I know reduced the code size.

Reply:
looks like only zephyr API which requires CONFIG_SYS_CLOCK_EXISTS is `sys_clock_disable()` which is called before jump to the application. Can do this optional - but probably the call is for something here,

Reply:
https://github.com/JuulLabs-OSS/mcuboot/blob/master/boot/zephyr/main.c#L77

Reply:
Can you then change that code to:

```
#ifdef(CONFIG_SYS_CLOCK_EXISTS)
sys_clock_disable();
#endif
```

Reply:
Yes - actually I was about asking whether this is good idea.